eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -34,6 +34,11 @@
           "OSVmImage": "MMS2019",
           "Pool": "azsdk-pool-mms-win-2019-general",
           "CMAKE_GENERATOR": "Visual Studio 16 2019"
+        },
+        "Win2022": {
+          "OSVmImage": "MMS2022",
+          "Pool": "azsdk-pool-mms-win-2022-general",
+          "CMAKE_GENERATOR": "Visual Studio 17 2022"
```

Comment:
This change doubles this part of the matrix by adding another 4 jobs. Perhaps we should keep a sparse matrix of win-2019 builds around.